corehq/apps/hqmedia/views.py

```diff
@@ -422,6 +422,15 @@ def valid_base_types(cls):
         return ['video']
 
 
+class ProcessTextFileUploadView(BaseProcessFileUploadView):
+    media_class = CommCareMultimedia
```

Comment:
I didn't write a new CommCareMultimedia class like CommCareText because there were three views (https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/hqmedia/_design/views) I'd need to rewrite, and I have it in my head that rewriting those unnecessarily is bad and causes long preindexes.

If that's not the case and I should write a CommCareText media class I'll definitely do that.